clang/include/clang/Basic/AArch64SVEACLETypes.def

```diff
@@ -201,6 +205,39 @@ SVE_OPAQUE_TYPE(__SVCount_t, __SVCount_t, SveCount, SveCountTy)
 
 SVE_SCALAR_TYPE(__mfp8, __mfp8, MFloat8, MFloat8Ty, 8)
 
+#ifndef NEON_VECTOR_TYPE
```

Comment:
Maybe leave comment these should not end up as `BuiltinType` (unlike all the others in this file).